lib/src/prefabs/modalUpload/ModalUploadHeader.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { Modal } from '../../components/modal/Modal';
+
+export function ModalUploadHeader({ onCloseClick }) {
+  return <Modal.Header headerText="Add files" onCloseClick={onCloseClick} />;
```

Comment:
💚 We could probably add a bit of customization through passing `headerText` as a prop and add `"Add files"` as the default value.